src/components/card/z-card-footer/index.tsx

```diff
@@ -26,38 +24,74 @@ export class ZCardFooter {
 
   @State() isOpen: boolean = false;
 
+  @State() allowTooltipAuthors: boolean = false;
+
+  private footer?: HTMLElement;
+  private ellipsisAuthors?: HTMLElement;
+
   @Listen("toggleClick")
   handleToggle(): void {
     this.isOpen = !this.isOpen;
   }
 
+  getTitleAuthors(): string {
+    return this.allowTooltipAuthors ? this.autori : "";
+  }
+
+  componentDidLoad() {
+    this.footer.ontransitionend = event => {
+      if (["visibility", "height"].includes(event.propertyName))
+        this.elementsEllipsis();
+    };
+  }
+
+  elementsEllipsis(): void {
+    this.allowTooltipAuthors = this.elementAuthorsHasEllipsis() ? true : false;
+  }
+
+  elementAuthorsHasEllipsis(): boolean {
+    return this.ellipsisAuthors.offsetWidth < this.ellipsisAuthors.scrollWidth;
+  }
+
   retrieveClass() {
     return {
       isopen: this.isOpen,
       real: this.cardtype === LicenseTypeEnum.real,
-      trial: this.cardtype === LicenseTypeEnum.trial,
+      trial: this.cardtype === LicenseTypeEnum.trial
     };
   }
 
   render() {
     return (
       <div class={this.faded && "faded"}>
-        <footer class={this.retrieveClass()}>
+        <footer
+          ref={el => (this.footer = el as HTMLElement)}
+          class={this.retrieveClass()}
+        >
           <span class="toggle">
             <slot name="toggle" />
           </span>
           <h2>{this.titolo}</h2>
           <div>
-            <p class="authors">
-              {this.autorilabel}: <b>{this.autori}</b>
+            <p
+              class="authors"
+              ref={el => (this.ellipsisAuthors = el as HTMLElement)}
+            >
+              <span title={this.getTitleAuthors()}>
+                <b>{this.autori}</b>
+              </span>
             </p>
             <p class="year_isbn">
               <span class="isbn">
-                ISBN (ed. cartacea): <b>{this.isbn}</b>
+                <span>
+                  <b>{this.isbn}</b> (ed. cartacea)
```

Comment:
Using the `<b>` tag prevents us from controlling the weight of the font with a design token. Better to use a span and the CSS `font-weight` property.